blockchain/stake/tickets.go

```diff
@@ -85,7 +85,10 @@ func (sn *Node) SpentByBlock() []chainhash.Hash {
 	return spent
 }
 
-// MissedByBlock returns the tickets that were missed in this block.
+// MissedByBlock returns the tickets that were missed in this block. This
```

Comment:
I wonder if this would be clearer if it were called `RemovedByBlock`, because that is what it's really referring to.